### DISCUSSION TITLE
Welcome to swn-character-creator Discussions!

### DISCUSSION THREAD

**Discussion by noah-sennett** [noah-sennett/swn-character-creator#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meisterlix** [comment]
Hi, first of all thanks for creating this character creator! 
The ability to build higher level character and maybe also a note space for inventory possibly including a stowed/readied toggle would be great! 
I hope you have a great day!
